frontend/pages/[gameId]/leaderboard.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from 'next/image';
+import { Inter } from 'next/font/google';
+
+import MatchMediaWrapper from '@/components/MatchMediaWrapper';
+import LeaderboardEntry from '@/components/LeaderboardEntry';
+
+import logo2 from 'public/logo2.png';
+
+const inter = Inter({ subsets: ['latin'] });
+
+const leaderboardEntries = [
+	{
+		name: 'Player 1',
+		score: 2000,
+		positionChange: 1,
+		isSelf: false,
+	},
+	{
+		name: 'Player 2',
+		score: 1500,
+		positionChange: -1,
+		isSelf: true,
+	},
+	{
+		name: 'Player 3',
+		score: 1100,
+		positionChange: 0,
+		isSelf: false,
+	},
+	{
+		name: 'Player 4',
+		score: 800,
+		positionChange: 2,
+		isSelf: false,
+	},
+	{
+		name: 'Player 5',
+		score: 700,
+		positionChange: -1,
+		isSelf: false,
+	},
+	{
+		name: 'Player 6',
+		score: 700,
+		positionChange: -1,
+		isSelf: false,
+	},
+];
+
+export default function Home() {
```

Comment:
This is a super small nitpick, but the function should be named relevantly (i.e. LeaderboardPage) just for clarity.

Reply:
Oof, I managed to neglect that for the other pages too. Thanks for pointing that out!